1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,19 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  // TODO
+  let arr = [];
+  let i = 0;
+  while (i % 2 === 0 && n > 0 && n > arr.length) {
+    arr.push(i);
+    i += 2;
+   
+  }
+  
+  return arr.toString(); // changes the numbers to strings
```

Comment:
```
function evenNumbers(n) {
  let counter = 0;
  let evenNum = 0;
  while (counter < n) {
    console.log(evenNum);
    evenNum += 2
    counter++;
  }
}
```
The above prints every value on a different line. As an exercise let do this to print them as a comma separated string:
1. Keep track of all the even numbers in the loop (hint: use array)
2. After loop, console.log the array using the `join`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join